### ISSUE TITLE
If id is removed from dom or not available Xarrow must give error 

### ISSUE THREAD

**Issue by vikasacharya16** [Eliav2/react-xarrows#135]
```
<YourComponent id={'comp1'}/>
<YourComponent id={'comp2'}/>
<Xarrow start={'comp1'} end={'comp2'}/>

```
if `comp2` is remove from `dom` Then I was getting like below. Instead Xarrow show give an error or something to handle this situation. Also When i put `comp2` back Xarrow is not rerendering. Kindly fix the issue

![chrome-capture](https://user-images.githubusercontent.com/32786586/163343401-3f9736d1-e09c-4ff8-8aab-a640a5aaf53e.jpg)

**Comment by gbarkhatov** [comment]
Did you solve the issue @vikasacharya16 ?

**Comment by viktormarinho** [comment]
Any updates on this @vikasacharya16 ?

**Comment by vikasacharya16** [comment]
No, I didn't solved. if the dom element start or end is removed then relation should vanish. if its changed it should rerender. give me some hints on how to achieve this.

**Comment by MagnusPladsen** [comment]
I have same issue here. 

If the target is removed from the DOM the arrow bugs out.

Since the arrows is a generated svg there is no possible way to remove it in the code. (as i know of)

<img width="427" alt="image" src="https://github.com/Eliav2/react-xarrows/assets/93226629/ef904236-40c9-442e-9ebe-b8b739a5a925">
